src/libs/actions/PaymentMethods.ts

```diff
@@ -28,61 +37,65 @@ function continueSetup() {
 }
 
 function openWalletPage() {
-    const onyxData = {
-        optimisticData: [
-            {
-                onyxMethod: Onyx.METHOD.MERGE,
-                key: ONYXKEYS.IS_LOADING_PAYMENT_METHODS,
-                value: true,
-            },
-        ],
-        successData: [
-            {
-                onyxMethod: Onyx.METHOD.MERGE,
-                key: ONYXKEYS.IS_LOADING_PAYMENT_METHODS,
-                value: false,
-            },
-        ],
-        failureData: [
-            {
-                onyxMethod: Onyx.METHOD.MERGE,
-                key: ONYXKEYS.IS_LOADING_PAYMENT_METHODS,
-                value: false,
-            },
-        ],
-    };
-
-    return API.read('OpenPaymentsPage', {}, onyxData);
-}
-
-/**
- *
- * @param {Number} bankAccountID
- * @param {Number} fundID
- * @param {Object} previousPaymentMethod
- * @param {Object} currentPaymentMethod
- * @param {Boolean} isOptimisticData
- * @return {Array}
- *
- */
-function getMakeDefaultPaymentOnyxData(bankAccountID, fundID, previousPaymentMethod, currentPaymentMethod, isOptimisticData = true) {
-    const onyxData = [
+    return API.read(
```

Comment:
API migration is not merged yet, please extract `optimisticData`, `successData` and `failureData` to this format:
```ts
    const optimisticData: OnyxUpdate[] = [...];
    const successData: OnyxUpdate[] = [...];
    const failureData: OnyxUpdate[] = [...];

    API.write('', parameters, {optimisticData, successData, failureData});
```

This will make it more type safe.

Reply:
Same thing for other functions, thanks!

Reply:
Done!